lib/dev/ui/prompt.rb

```diff
@@ -88,11 +88,25 @@ def ask(question, options: nil, default: nil, is_file: nil, allow_empty: true)
         #
         def confirm(question)
           puts_question("#{question} {{yellow:(choose with ↑ ↓ ⏎)}}")
-          InteractivePrompt.call(%w(yes no)) == 'yes'
+          interactively_ask(question, %w(yes no)) == 'yes'
         end
 
         private
 
+        def interactively_ask(question, options)
```

Comment:
I'm not a huge fan of this name in the context of an `ask` method which is interactive by virtue of calling this.  With the suggested DRYing above, this could be inlined into ask.